packages/ozone/src/auth-verifier.ts

```diff
@@ -113,9 +108,30 @@ export class AuthVerifier {
     }
     const payload = await verifyJwt(jwtStr, this.serviceDid, getSigningKey)
     const iss = payload.iss
-    const isAdmin = this.admins.includes(iss)
-    const isModerator = isAdmin || this.moderators.includes(iss)
-    const isTriage = isModerator || this.triage.includes(iss)
+
+    // Always allow serviceDid to access as admin
+    if (iss === this.serviceDid) {
+      return {
+        credentials: {
+          type: 'standard',
+          iss,
+          aud: payload.aud,
+          isAdmin: true,
+          isModerator: true,
+          isTriage: true,
+        },
+      }
+    }
```

Comment:
Perhaps we could avoid this exception if we folded the account into `seedInitialMembers()`.

Reply:
You know, the more I think about it we may want to keep the deletion guard but not add any other exceptions for the service DID.  I think there could be legitimate reasons for a deployment to not want to make the service DID a team member.

For the Ozone distribution we configure the service DID to be an admin anyway, so most users will have this by default.  Perhaps the deletion rule should be to ensure you don't delete or demote yourself.  That should be enough to ensure there's no way to lock yourself out.